pkg/ccl/backupccl/testdata/backup-restore/schedule-privileges

```diff
@@ -122,21 +122,10 @@ exec-sql user=testuser
 DROP SCHEDULE $fullID
 ----
 
-query-sql
-with schedules as (SHOW SCHEDULES FOR BACKUP) SELECT label,command,owner FROM schedules WHERE id IN
-($fullID, $incID) ORDER BY next_run;
-----
-foocluster_admin BACKUP INTO LATEST IN 'external://foo/cluster' WITH detached root
-
 exec-sql
 REVOKE ADMIN FROM testuser
 ----
 
-exec-sql expect-error-regex=(must be admin or owner of the schedule [0-9]+ to DROP it) user=testuser
```

Comment:
this seems like a case we want to still cover. can you think of a way to still cover it?